lib/utils/config/definitions.js

```diff
@@ -1624,7 +1624,7 @@ define('provenance', {
   default: false,
   type: Boolean,
   description: `
-    Indicates that a provenance statement should be generated.
+    When you publish from a supported cloud CI/CD system, this will publicly establish where the package was built and published from.
```

Comment:
```suggestion
    When you publish from a supported cloud CI/CD system, this will publicly
    establish where the package was built and published from.
```